api-reference/10 UI Widgets/dxGantt/1 Configuration/validation/enableDependencyValidation.md

```diff
@@ -5,7 +5,74 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Enables [task dependencies](/api-reference/10%20UI%20Widgets/dxGantt/1%20Configuration/dependencies '/Documentation/ApiReference/UI_Widgets/dxGantt/Configuration/dependencies/') validation.
```

Comment:
`validateDependencies` may be a better name for this option, since it is shorter and do not repeat the word "validation", which is the name of the object that nests this option.

Reply:
I'll discuss this with the team.